BAZProject/Views/MainViews/NewMovies/CollectionViewCells/UpcomingCollectionViewCell.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UpcomingCollectionViewCell.swift
+//  BAZProject
+//
+//  Created by Mario Arceo on 01/03/23.
+//
+import UIKit
+
+class UpcomingCollectionViewCell: UICollectionViewCell {
+    
+    let identifier = "UpcomingCustomCell"
```

Comment:
Para no crear una instancia de esta clase es mejor poner el identifier con constante estática.
```suggestion
    static let identifier = "UpcomingCustomCell"
```